vars/grafanaAgent.groovy

```diff
@@ -0,0 +1,46 @@
+boolean supported() {
+ //   int exitCode = (sh (returnStatus: true, script: 'systemctl list-units --all | grep grafana-agent')) as int
+    int exitCode = 0
+    return exitCode == 0
+}
```

Comment:
We need to support agents where grafana-agent is not installed. It may help a lot to avoid the issues with pipelines especially on the beginning of this migration